### PR TITLE
Mark more interface method returns with `#[ReturnTypeWillChange]` for PHP 8.1 compatibility

### PR DESCRIPTION
**Pull request by Crell**
Some return type flags got missed for PHP 8.1.  Adding them now, to clean up my logs. :smiley: 

|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | yes
| BC Break      | no
| New Feature   | o
| RFC           | no
| QA            | no

### CODE REVIEW

File: src/Reader/AbstractFeed.php
Comment (froschdesign) [inline src/Reader/AbstractFeed.php:9]:
```suggestion
// phpcs:ignore SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse
use ReturnTypeWillChange;
```

**Comment by Crell** [comment]
Ugh.  The hard-requirement on signing every individual commit breaks editing through GitHub suggestions.  That's lovely.  I don't know how you want this sorted out, but I leave it to the maintainers.

**Comment by Ocramius** [comment]
Meh, it's stupid corporate vogonism from the USA: ignore it.

**Review by Ocramius — APPROVED** [review]

**Review by Ocramius — APPROVED** [review]
:ship: thanks @Crell!

**Pull request merged**
